composer.json

```
@@ -12,10 +12,10 @@
     "require": {
         "php": ">=7.0",
         "ext-json": "*",
-        "elasticsearch/elasticsearch": "^5.1.0||^6.0.0"
+        "elasticsearch/elasticsearch": "^5.1.0||^6.0.0||^7.11"
```

```suggestion
        "elasticsearch/elasticsearch": "^5.1||^6.0||^7.0"
```

except if versions between `7.0.0` and `7.11` are really broken?

---

PHP8 is only supported from `^7.11` but you're right, there's no reason to skip the `7.0`

I updated the branch